lib/tasks/launch-browserstack-browser.js

```diff
@@ -72,8 +48,39 @@ let argv = require('yargs')
   .option('build', {
     describe: 'Build Name, defaults to process.env.TRAVIS_JOB_NUMBER or process.env.BITBUCKET_BUILD_NUMBER, if set, prefixed with process.env.BROWSERSTACK_BUILD_NAME_PREFIX, if set',
     default: build
+  })
+  .option('project', {
```

Comment:
Can you show an example of how this looks in the BrowserStack UI? I was never able to get this option to work.

Reply:
<img width="314" alt="Screenshot 2019-09-06 at 17 54 59" src="https://user-images.githubusercontent.com/659945/64437722-7ae2c380-d0cf-11e9-8a66-babdf4ac3b62.png">

Like this.

Reply:
Obviously you can filter by project and so on. I locally linked this package to my project and did out with actual test run and worked just fine. It is anyway optional parameter in this PR, so 100% backward compatible.